plantuml diagrams not rendering

running `./mvnw` I get the following error:

```
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 40 resources
[INFO] Rendered /home/edewit/Downloads/app-labels-master/README.adoc
[INFO] Rendered /home/edewit/Downloads/app-labels-master/index.adoc
java.io.IOException: Cannot run program "/opt/local/bin/dot": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at java.base/java.lang.Runtime.exec(Runtime.java:635)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.startThreads(ProcessRunner.java:163)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.runJob(ProcessRunner.java:123)
	at net.sourceforge.plantuml.api.TimeoutExecutor$MyThread.run(TimeoutExecutor.java:79)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:339)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:270)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 5 more
java.io.IOException: Cannot run program "/opt/local/bin/dot": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at java.base/java.lang.Runtime.exec(Runtime.java:635)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.startThreads(ProcessRunner.java:163)
	at net.sourceforge.plantuml.cucadiagram.dot.ProcessRunner$MainThread.runJob(ProcessRunner.java:123)
	at net.sourceforge.plantuml.api.TimeoutExecutor$MyThread.run(TimeoutExecutor.java:79)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:339)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:270)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 5 more
[INFO] Rendered /home/edewit/Downloads/app-labels-master/labels-annotation-for-openshift.adoc
```
Do I need to install something specific for the diagrams to work?

![image](https://user-images.githubusercontent.com/51133/65023424-14fa0580-d933-11e9-837b-973b0c496f98.png)

--- a/README.adoc
+++ b/README.adoc
@@ -9,6 +9,8 @@ run but they are highly recommended.
 
 Convert the AsciiDoc to PDF and HTML using Asciidoctor by invoking the `process-resources` goal (configured as the default goal):
 
- $ mvn
+ $ ./mvnw
+
+IMPORTANT: You need to install `graphviz` for the diagrams to show.
 
 Open the file _target/generated-docs/labels-annotation-for-openshift.pdf_ or _target/generated-docs/labels-annotation-for-openshift.html_

